app/selectors/views.js

```diff
@@ -40,7 +40,7 @@ export const getThreadDraft = createSelector(
 );
 
 export function getProfileImageUri(state) {
-    return state.views.user.profileImageUri;
+    return state.views?.user?.profileImageUri;
```

Comment:
What changed to require so much safety-checking in this specific case?

Reply:
rehydrating after refreshing (dev mode) was causing this to crash -> very annoying while developing